test/distributed/tensor/test_random_ops.py

```diff
@@ -506,11 +506,13 @@ def test_deterministic_uniform_2d(self):
                     shard_dim = placement.dim
                     local_shard_list_on_dim[shard_dim] = []
                     for shard_idx_on_dim in range(mesh_dim_size):
-                        shard_size, shard_offset = placement._local_shard_size_on_dim(
+                        (
+                            shard_size,
+                            shard_offset,
+                        ) = placement._local_shard_size_and_offset(
```

Comment:
curious: I guess this comes from VS code autoformat but is not a suggestion from pytorch linter. What's our policy here? Accept it only if the linter doesn't complain?

Reply:
hmm, i don't think i have enabled vscode linter, i just use 'lintrunner' command in terminal to perform formatting.  If something is not matching expectation, i might need to update my lintrunner version again (but i updated it sometime last week)